.github/workflows/codeql-scan.yml

```
@@ -22,8 +22,20 @@ on:
     branches:
       - master
       - main
+    paths:
+      - '.github/workflows/codeql-scan.yml'
+      - '**/pom.xml'
+      - 'fesod/**'
+      - 'fesod-*/**'
+      - '!**.md'
```

```suggestion
      - "**.java"
```
Only java file to trigger ci.

---

Hi @GOODBOY008, Thanks for the suggestion! Just wondering, would `"**.java"` actually cover everything we need here? From what I can see, it would only match Java source files, but we also have test resources like Excel and CSV files in `src/test/resources/` that our tests depend on. If someone updates those test files, we probably want CI to run and make sure the tests still pass. Also, `"**.java"` wouldn't catch changes to pom.xml or the CI workflow itself and it would possibly still have the same `mvnw` issue you pointed out earlier. Just wondering if we just add `mvnw` on top of the existing list to cover the Maven wrapper case, would this actually be better?